bash-script-inventory.md

```diff
@@ -1,4 +1,4 @@
-# Getting a inventory in repositories with Bash
+# Getting an Inventory in Repositories with Bash
```

Comment:
It looks like using title casing goes against this repo's norms based on the [README.md](https://github.com/cisco-open/inclusive-language#inclusive-language-resources-and-tool-examples). Consider reverting this change.